samples/net/wpan_serial/README.rst

```diff
@@ -26,8 +28,47 @@ Building and Running
 
      $ sudo systemctl disable ModemManager.service
 
-#. Build and flash the sample Zephyr application to a board
-   with a 802.15.4 radio and USB controller
+#. Build the sample Zephyr application to a board with a 802.15.4 radio
+   and USB controller. There are configuration files for various setups
+   in the ``samples/net/wpan_serial`` directory:
+
+   - :file:`prj.conf`
+     This is the standard default config. This can be used by itself for
+     hardware which has native 802.15.4 support.
+
+   - :file:`overlay-cc2520.conf`
+     This overlay enables support for CC2520 transceiver
+
+   - :file:`overlay-rf2xx.conf`
+     This overlay enables support for RF2XX transceiver
+
+   To build the wpan_serial sample:
+
+   .. zephyr-app-commands::
+     :zephyr-app: samples/net/wpan_serial
+     :board: <board name>
+     :conf: "prj.conf [overlay-<RADIO>.conf]"
+     :goals: build
+     :compact:
+
+   Here's how to build the sample for the Atmel SAM R21 Xplained Pro
+   Development Kit. Note that for this SoC, you don't need to include
+   ``overlay-rf2xx.conf``.
+
+   .. zephyr-app-commands::
+     :zephyr-app: samples/net/wpan_serial
+     :board: atsamr21_xpro
+     :conf: "prj.conf"
+     :goals: build
+     :compact:
+
+   Flash your board
+   .. zephyr-app-commands::
+     :zephyr-app:
```

Comment:
Documentation error seems to be related to this section, you probably need to fill these couple of empty values